airflow/migrations/versions/8646922c8a04_change_default_pool_slots_to_1.py

```diff
@@ -26,48 +26,21 @@
 
 import sqlalchemy as sa
 from alembic import op
-from sqlalchemy import Column, Integer, String
-from sqlalchemy.ext.declarative import declarative_base
-
-from airflow.utils.sqlalchemy import UtcDateTime
 
 # revision identifiers, used by Alembic.
 revision = '8646922c8a04'
 down_revision = '449b4072c2da'
 branch_labels = None
 depends_on = None
 
-Base = declarative_base()
-BATCH_SIZE = 5000
-
-
-class TaskInstance(Base):  # type: ignore
-    """Minimal model definition for migrations"""
-
-    __tablename__ = "task_instance"
-
-    task_id = Column(String(), primary_key=True)
-    dag_id = Column(String(), primary_key=True)
-    execution_date = Column(UtcDateTime, primary_key=True)
-    pool_slots = Column(Integer, default=1)
-
 
 def upgrade():
     """Change default pool_slots to 1 and make pool_slots not nullable"""
-    connection = op.get_bind()
-    sessionmaker = sa.orm.sessionmaker()
-    session = sessionmaker(bind=connection)
-
-    session.query(TaskInstance).filter(TaskInstance.pool_slots.is_(None)).update(
-        {TaskInstance.pool_slots: 1}, synchronize_session=False
-    )
-    session.commit()
-
     with op.batch_alter_table("task_instance", schema=None) as batch_op:
-        batch_op.alter_column("pool_slots", existing_type=sa.Integer, nullable=False)
+        batch_op.alter_column("pool_slots", existing_type=sa.Integer, nullable=False, server_default='1')
```

Comment:
Should this use `text()` as well? (Honestly I’m not sure about the difference it makes.)

Reply:
It works the same as using text. There's actually no need of using text since we are not giving it SQL expression https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.alter_column.params.server_default